explorer/interpreter/value_transform.h

```diff
@@ -159,6 +170,9 @@ class ValueTransform : public TransformBase<Derived> {
   auto operator()(Nonnull<const Element*> elem) -> Nonnull<const Element*> {
     return TransformDerived<Nonnull<const Element*>>(elem);
   }
+
+  // Preserve vptr during transformation.
+  auto operator()(Nonnull<const VTable** const> vptr) -> auto{ return vptr; }
```

Comment:
```suggestion
  auto operator()(Nonnull<const VTable** const> vptr) -> Nonnull<const VTable** const> { return vptr; }
```

I don't think you should do `auto ... -> auto`, it feels redundant. Repeating seems consistent with above, we do tend to be explicit about return types.

Reply:
My bad, bad copy/paste.